src/liquid/components/ld-card/ld-card-stack/ld-card-stack.css

```diff
@@ -0,0 +1,78 @@
+:host,
+.ld-card-stack {
+  --ld-card-stack-translate-amount-x: var(--ld-sp-4);
+  --ld-card-stack-translate-amount-y: var(--ld-sp-4);
+  --ld-card-stack-translate-direction-x: -1;
```

Comment:
Why not also add the option to change the y direction, as well?

Reply:
I've talked to Nic and he said he would only need the three options.